src/AdminController.js

```diff
@@ -153,6 +172,30 @@ class AdminController extends EditController {
     let opts = {type: this.uploadType, ...this.uploadOptions}
     return dataManager.importFileToModel(this.modelIdentity, path, opts)
   }
+
+  _download (req, res) {
+    return Promise.all([
+      this.model.find(),
+      this._modelAttributes(),
+    ]).spread((records, attrs) => {
+      let dateAttrs = attrs.filter(x => x.type=='datetime' || x.type=='date')
+      records = records.map((x) => {
+        x = _.pick(x, this.displayFields)
+        dateAttrs.forEach((f) => {
+          if (x[f.name]) {
+            x[f.name] = moment(x[f.name]).format()
+          }
+        })
+        return x
```

Comment:
would be nice to have a place here to inject custom formatting by over-riding a do-nothing transformation class.